storybookjs/storybook
# Build static storybooks utility script

Issue: N/A

## What I did

To run chromatic on local machine, you should build all the static storybooks first

## How to test

```
yarn build-storybooks
yarn chromatic
```

Is this testable with Jest or Chromatic screenshots?
Does this need a new example in the kitchen sink apps?
Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
